logstash-core-event-java/src/main/java/com/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -233,11 +215,7 @@ public IRubyObject ruby_sprintf(ThreadContext context, IRubyObject format) throw
             try {
                 return RubyString.newString(context.runtime, event.sprintf(format.toString()));
             } catch (IOException e) {
-                throw new RaiseException(
-                    getRuntime(),
-                    (RubyClass) getRuntime().getModule("LogStash").getClass("Error"),
-                    "timestamp field is missing", true
-                );
+                throw new RaiseException(getRuntime(), (RubyClass)getRuntime().getModule("LogStash").getClass("Error"), "timestamp field is missing", true);
```

Comment:
Consider making a LogStashError utility class for all others to use. Along the lines of:
```
import org.jruby.Ruby;
import org.jruby.RubyClass;
import org.jruby.RubyException;
import org.jruby.anno.JRubyClass;
import org.jruby.exceptions.RaiseException;

@JRubyClass(name = "JrJackson::ParseError", parent = "RuntimeError")
public class ParseError {

    public static RaiseException newParseError(Ruby ruby, String message) {
        RubyClass errorClass = ruby.getModule("JrJackson").getClass("ParseError");
        return new RaiseException(RubyException.newException(ruby, errorClass, message), true);
    }
}
```